netbox/templates/dcim/device.html

```diff
@@ -286,6 +286,31 @@ <h5 class="card-header">
             </div>
             {% include 'inc/panels/contacts.html' %}
             {% include 'inc/panels/image_attachments.html' %}
+
+            <div class="text-end mb-4">
+              <select class="btn btn-sm btn-outline-dark rack-view">
+                <option value="images-and-labels" selected="selected">Images and Labels</option>
+                <option value="images-only">Images only</option>
+                <option value="labels-only">Labels only</option>
+              </select>
```

Comment:
Not sure we need this toggle for the device view, but if we want to keep it we should move the whole thing into a separate inclusion template to avoid duplicate template code. My vote would be to just omit the selector though.